### PR TITLE
Add indicator of which state was featured in the most recent batch.

### PR DESCRIPTION
**Pull request by fractionalhare**
This is intended to resolve issue #161.

Screenshot of new functionality on HTML page. 
Screen Shot 2020-11-06 at 8 49 00 PM
Screenshot of new functionality on .txt file:
Screen Shot 2020-11-06 at 8 48 10 PM

### CODE REVIEW

**Comment by fractionalhare** [comment]
Screen Shot 2020-11-06 at 9 01 01 PM

showing agreement with most recent commit

File: print-battleground-state-changes [hunk 1]
```diff
@@ -374,7 +374,7 @@ html_chunks = []
 batch_time = max(itertools.chain.from_iterable(summarized.values()), key=lambda s: s.timestamp).timestamp
 print(tabulate([
     ["Last updated:", scrape_time.strftime("%Y-%m-%d %H:%M UTC")],
-    ["Latest batch received:", batch_time.strftime("%Y-%m-%d %H:%M UTC")],
+    ["Latest batch received:", batch_time.strftime("%Y-%m-%d %H:%M UTC"), states_updated[-1]],
```
Comment (thomasdesr) [inline print-battleground-state-changes]:
```suggestion
    [f"Latest batch received ({states_updated[-1]}):", batch_time.strftime("%Y-%m-%d %H:%M UTC")],
```

Comment (thomasdesr) [reply print-battleground-state-changes]: Propose we merge this into so we don't get the extra column.

**Comment by fractionalhare** [comment]
after adding in @thomasdesr changes, the screenshot on the txt file is

Screen Shot 2020-11-06 at 9 14 18 PM

**Comment by fractionalhare** [comment]
merging with a LGTM from @bojanrajkovic :)

**Pull request merged**